static/src/javascripts/projects/common/utils/detect.js

```diff
@@ -435,16 +435,20 @@ define([
 
     function adblockInUse() {
         var displayed = '',
-            isAdblock = false;
+            isAdblock = false,
+            mozBinding = '',
+            mozBindingHidden = -1;
 
         $.create('<div class="ad_unit"></div>').appendTo(document.body);
         displayed = $('.ad_unit').css('display');
+        mozBinding = $('.ad_unit').css('-moz-binding');
+        if (mozBinding !== undefined) {
```

Comment:
I think that:
'if(typeof variable_here === 'undefined'){}'
is better to detect undefined